src/fparser/tests/test_Fortran2003.py

```diff
@@ -84,13 +84,13 @@ def test_Program(): # R201
         cls = Program
         reader = get_reader('''\
       subroutine foo
-      end subroutine foo
+      end subroutine Foo
```

Comment:
This has been modified to upper case. I'm not sure why unless it is to test one of the case independent comparisons, or perhaps to check that the case does not change from input to output.